search-commons/src/main/java/no/unit/nva/search/resource/ContributorNodeFilter.java

```diff
@@ -0,0 +1,57 @@
+package no.unit.nva.search.resource;
+
+import static no.unit.nva.constants.Words.AFFILIATIONS;
+import static no.unit.nva.constants.Words.COUNTRY_CODE;
+import static no.unit.nva.constants.Words.DOT;
+import static no.unit.nva.constants.Words.ENTITY_DESCRIPTION_CONTRIBUTORS_PATH;
+import static no.unit.nva.constants.Words.IDENTITY;
+import static no.unit.nva.constants.Words.NO;
+import static no.unit.nva.constants.Words.VERIFICATION_STATUS;
+import static no.unit.nva.constants.Words.VERIFIED;
+
+import com.fasterxml.jackson.core.JsonPointer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+
+import no.unit.nva.search.common.records.JsonNodeMutator;
+
+public final class ContributorNodeFilter implements JsonNodeMutator {
+
+    static final int FIRST_FIVE_CONTRIBUTORS = 5;
```

Comment:
Should be renamed to either "FIVE", because thats what the value is. Or "MINIMUM_INCLUDED_CONTRIBUTERS", because thats what the value controlls. Right now if we wanted to change it to 6 we would need to rename both value and name